.github/workflows/cpu_test.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
   pytest:
     needs: setup
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-22.04-16core
```

Comment:
why is this needed?

Reply:
I am actually loading a llama-8b into memory in `torchprime/torch_xla_models/tests/test_load_model_from_hf.py` and seems it failed with OOM with ubuntu-22.04. It works in my local machine, so want to check if increase vm size can help.